actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -850,7 +850,7 @@ def hidden_field(object_name, method, options = {})
       #   # => <input type="file" id="user_avatar" name="user[avatar]" />
       #
       #   file_field(:post, :image, :multiple => true)
```

Comment:
if we are modifying these docs, we might as well change the hash syntax here to `multiple: true`